modules/jpa-envers/src/main/java/io/oasp/module/jpa/dataaccess/api/AdvancedRevisionEntiryOracle.java

```diff
@@ -0,0 +1,20 @@
+package io.oasp.module.jpa.dataaccess.api;
+
+import javax.persistence.Column;
+
+/**
+ * TODO vkiran This type ...
```

Comment:
I think this class should not be here. AFAIK it was created just for trying something out